web_responsive/static/src/js/web_responsive.js

```diff
@@ -276,6 +285,20 @@ odoo.define('web_responsive', function (require) {
         },
     });
 
+    BasicController.include({
+        
+        /**
+         * @override
+         */
+        canBeDiscarded: function (recordID) {
+            if (this.model.isDirty(recordID || this.handle)) {
+                $('.o_menu_apps .dropdown:has(.dropdown-menu.show) > a').dropdown('toggle');
+                $('.o_menu_sections li.show:has(.dropdown-menu.show) > a').dropdown('toggle');
```

Comment:
I know that is confuse.. but best "attack" the element that have "toggle='dropdown'" defined in the dataset.

``` $('.o_menu_sections li.show .dropdown-toggle').dropdown('toggle');```

Reply:
https://github.com/OCA/web/compare/044ccb31de238ccb90e302bc0fca9376f3bd777f..156d3bb5d69e5397223c6f63680716634ae00f40